spec/factories/offence_classes.rb

```diff
@@ -21,10 +21,10 @@
 
   trait :with_lgfs_offence do
     after(:create) do |record|
-      create :offence, :miscellaneous, offence_class: record
+      offence = create :offence, :miscellaneous, offence_class: record
+      offence.fee_schemes << (FeeScheme.first || create(:fee_scheme, :agfs_nine))
```

Comment:
Shouldn't the fee scheme here (`create(:fee_scheme, :agfs_nine))`) be for `LGFS` ? Also, `FeeScheme.first` should probably be `FeeScheme.lgfs.first` to return the first LGFS rather than the first one it finds (that could be LGFS).